services/movie_session.py

```diff
@@ -0,0 +1,39 @@
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie_id=movie_id,
+                                       cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: datetime = None) -> MovieSession:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)

```

Comment:
The `get_movies_sessions` function correctly uses `.filter()` to retrieve all sessions for a given date. Ensure that this logic is intended to handle multiple sessions on the same date, as using `.get()` would not be appropriate here.